docs/synthetic-tokens/long-short-pair.md

```diff
@@ -13,24 +13,9 @@ Some ideas for contracts that can be made with the LSP include:
 - Covered call options.
 - [Range Tokens for treasury management](https://medium.com/uma-project/treasury-diversification-with-range-tokens-145d4b12614e).
 
-## LSP Example
-
-Bob decides that he wants to offer a suite of covered call options to the DeFi community. As his first product, he has decided that he wants to create a covered call option on ETH. He knows that the ETH/USD price identifier has already been registered by UMA governance, so it should take him no more than a few minutes to create this contract.
-
-The price of ETH is currently 2500, so Bob deploys a covered call option, that uses UMA's [call options library](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/CoveredCallLongShortPairFinancialProductLibrary.sol), and sets the contract's strike to 3000 ETH/USD.
+## LSP Contract Deployment
```

Comment:
This section seems unnecessary since there is a separate LSP deployment tutorial. Can you remove this section and just replace with adding one sentence to the previous section linking to the deployment tutorial?

Reply:
I agree with this. Including a sentence in the section above and removing.